e2e/test/E2EMsTestBase.cs

```diff
@@ -6,24 +6,25 @@
 using System.Threading;
 using Microsoft.ApplicationInsights.DataContracts;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
 
 namespace Microsoft.Azure.Devices.E2ETests
 {
     /// <summary>
     /// This class creates an instance of the logger for each test and logs the test result along with other useful information.
     /// </summary>
-    public class E2ETestBase
+    public class E2EMsTestBase
```

Comment:
Calling this MsTestBase because we can have a mix of MsTest and XUnit. This should help differentiate.